### ISSUE TITLE
As a user I am able to search through my emails

### ISSUE THREAD

**Issue by jasmeetakaur** [jkaurs/jmail#8]

**Comment by mrjaspsing** [comment]
@jasmeetakaur Thanks for creating these epics. We need to get started on the sprint and release planning as well. 
Here are some links that we need to go through ...
https://www.agile42.com/documents/22/Scrum_in_a_nutshell_t69WC2n.pdf
https://image.slidesharecdn.com/scrumcheatsheet-13359759117381-phpapp01-120502112840-phpapp01/95/scrum-cheat-sheet-1-728.jpg?cb=1335958136
https://vitalitychicago.com/wp-content/uploads/2017/05/agile-scrum-tip-sheet-sept-2018.png
https://www.forsa-advies.nl/doc/folders/scrum_quick_reference_card_forsa_advies_uk.pdf
https://www.dummies.com/careers/project-management/agile-project-management-for-dummies-cheat-sheet/

**Comment by jasmeetakaur** [comment]
Was that the only amount of work to be done on that?
I didn’t assign all the issues to you as it was quite redundant.

**Comment by mrjaspsing** [comment]
@jasmeetakaur Correct. That was all. You created all epics and stories/issues. Good job.
We normally do not assign issues/stories yet. We do that in what is called a "Sprint Planning" meeting.
In that meeting we decide - WHAT will need to be done in the upcoming sprint and WHO will do it.
